.drone.yml

```diff
@@ -0,0 +1,73 @@
+kind: pipeline
+name: default
+
+steps:
+- name: test
+  image: golang:1.12-stretch
+  commands:
+  - apt update -qq
+  - apt install -qqy jq
+  - go test -v ./...
+
+---
+
+kind: pipeline
+name: reachability-test
+
+steps:
+- name: build
+  image: golang:1.12-stretch
+  commands:
+  - go build ./jiup/rules/reachability-test
+- name: test
+  image: debian:stretch
+  commands:
+  - apt update -qq
+  - apt install -qqy jq ca-certificates
+  - ./reachability-test --alternate-output
+# todo: decide on conditions to run this and updating the registry
+
+depends_on:
+- default
+
+---
+
+kind: pipeline
+name: update-registry
+
+steps:
+- name: clone-registry
+  image: golang:1.12-stretch
+  commands:
+  - git clone https://github.com/just-install/registry
+- name: build
+  image: golang:1.12-stretch
+  commands:
+  - go build
+- name: update
+  image: debian:stretch
+  commands:
+  - apt update
+  - apt install -qqy jq ca-certificates
+  - cd registry
+  - ../just-install-updater-go -c message.txt just-install.json
+  depends_on: [ build, clone-registry ]
+- name: commit
+  image: golang:1.12-stretch
+  commands:
+  - cd registry
+  - git config user.name "just-install-bot"
+  - git config user.email "just-install-bot@outlook.com"
+  - git add just-install.json
+  - cat message.txt
+  - git commit -F message.txt || true
+  depends_on: [ update ]
+#-name: push
+# image: golang:1.12-stretch
+# commands:
+# - cd registry
+# - git push "https://$GITHUB_TOKEN@github.com/just-install/registry.git" master >/dev/null 2>/dev/null
+# depends_on: [ commit ]
```

Comment:
I'll enable this once the PR is ready. Also, note to self about adding conditions to this.